website/server/controllers/api-v3/pushNotifications.js

```diff
@@ -0,0 +1,108 @@
+import { authWithHeaders } from '../../middlewares/api-v3/auth';
+import _ from 'lodash';
+import {
+  BadRequest,
+  NotAuthorized,
+} from '../../libs/api-v3/errors';
+
+let api = {};
+
```

Comment:
@vIiRuS can you confirm if that was your intention with the `GET`? If it was not, then it should definitely be a `DELETE`